addons/l10n_jo_edi/models/account_edi_xml_ubl_21_jo.py

```diff
@@ -406,7 +409,7 @@ def _export_invoice_vals(self, invoice):
             'accounting_customer_party_vals': {
                 'party_vals': self._get_empty_party_vals() if is_refund else self._get_partner_party_vals(customer, role='customer'),
                 'accounting_contact': {
-                    'telephone': '' if is_refund else invoice.partner_id.phone or invoice.partner_id.mobile,
+                    'telephone': self._sanitize_phone('' if is_refund else invoice.partner_id.phone or invoice.partner_id.mobile),
```

Comment:
Let's avoid calling `_sanitize_phone` with an empty string.
```suggestion
                    'telephone': '' if is_refund else self._sanitize_phone(invoice.partner_id.phone or invoice.partner_id.mobile),